testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvMultinodeSnowflakeWithPrivateKey.java

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Why do we add a product test instead of integration test? https://trino.io/docs/current/develop/tests.html#avoid-product-tests

Reply:
The product test was added for parity with existing testing behaviour of Snowflake Suite. Snowflake credentials testing does require a snowflake server too. In that case, this would make sense to be added.